app/lib/importer/base_importer.rb

```diff
@@ -68,8 +68,8 @@ def clean_up!
 
   protected
 
-  def in_work_unit(*args, &block)
-    work_unit = Concurrent::Promises.future_on(@executor, *args, &block)
+  def in_work_unit(...)
+    work_unit = Concurrent::Promises.future_on(@executor, ...)
```

Comment:
This originally had a bad fix from https://github.com/rubocop/rubocop/issues/12087, but manually changed fixed it